server/controllers/transactionController.js

```diff
@@ -99,20 +109,26 @@ class TransactionController {
     */
 
   static async getSingleTransactionById(req, res) {
-    const { param } = req.body; const id = param;
+    const { param, loggedinUser } = req.body; const id = param;
     let transaction;
 
     try {
+      if (loggedinUser.type === 'client') {
+        const checkForOnwer = await pool.query(queries.join.getTransactionById,
+          [loggedinUser.id, id]);
+        if (!checkForOnwer.rows[0]) { return util.errorstatus(res, 400, 'Account Transaction does not belong to this User'); }
+      }
+
```

Comment:
If statements should be broken to a seperate line if it has curly brackets for consistency in the codebase.